packages/common/src/store/reachability/reducer.ts

```diff
@@ -12,11 +12,11 @@ const initialState = {
 const reducer = createReducer<ReachabilityState, ReachabilityActions>(
   initialState,
   {
-    [actions.SET_REACHABLE]() {
-      return { networkReachable: true }
+    [actions.SET_REACHABLE](state: ReachabilityState) {
+      return { ...state, networkReachable: true }
     },
-    [actions.SET_UNREACHABLE]() {
-      return { networkReachable: false }
+    [actions.SET_UNREACHABLE](state: ReachabilityState) {
+      return { ...state, networkReachable: false }
```

Comment:
Should I have included this to begin with? I don't see where it's being used in this change

Reply:
yeah I think this is just safer. I had added a prop here and then removed it, but forgot to spread current state when I did that, which is kinda scary.

Reply:
though typechecking does help. let's chat at client sync about whether this is part of styleguide or not!